scripts/migrate_piwik_historical_data.py

```diff
@@ -0,0 +1,152 @@
+import MySQLdb
+from socket import inet_ntoa
+from binascii import b2a_hex
+import pprint
+from keen import KeenClient
+import urlparse
+
+def main():
+
+    keen_client = KeenClient(
```

Comment:
Can you please document any external dependencies that need to be installed in the docstring for this script?